inc/signup_notification/namespace.php

```diff
@@ -160,3 +185,340 @@ function altis_signup_blog_notification( $domain, $path, $title, $user_login, $u
 
 	return true;
 }
+
+/**
+ * Sends a confirmation request email to a user when they sign up for a new user account (without signing up for a site
+ * at the same time). The user account will not become active until the confirmation link is clicked.
+ *
+ * This is the notification function used when no new site has
+ * been requested.
+ *
+ * Filter {@see 'wpmu_signup_user_notification'} to bypass this function or
+ * replace it with your own notification behavior.
+ *
+ * Filter {@see 'wpmu_signup_user_notification_email'} and
+ * {@see 'wpmu_signup_user_notification_subject'} to change the content
+ * and subject line of the email sent to newly registered users.
+ *
+ * @since MU (3.0.0)
+ *
+ * @param string $user_login The user's login name.
+ * @param string $user_email The user's email address.
+ * @param string $key        The activation key created in wpmu_signup_user()
+ * @param array  $meta       Optional. Signup meta data. Default empty array.
+ * @return bool
+ */
+function altis_signup_user_notification( $user_login, $user_email, $key, $meta = [] ) {
```

Comment:
I'm guessing the standard WordPress functions had to be replicated because they don't call the filter `wp_mail_from`.

Reply:
Yeah if you look at the code that was already in this file that’s what it was doing, we just missed these other core notifications at the time so for a lot of stacks these emails will never successfully send if the from address doesn’t match the approved sending domains.